### PR TITLE
refactor(condo): DOMA-10204 rename FindOrganizationsForAddress

### PR DESCRIPTION
**Pull request by vovaaxeapolla**
rename FindOrganizationsForAddress to FindOrganizationsByAddress

### CODE REVIEW

File: apps/condo/domains/resident/schema/FindOrganizationsByAddressService.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ const {
     getOrganizationsWithAcquiring,
     getOrganizationsWithMeters,
     getBillingInformationForOrganizations,
-} = require('@condo/domains/resident/utils/serverSchema/findOrganizationsForAddress')
+} = require('@condo/domains/resident/utils/serverSchema/FindOrganizationsByAddress')
```
Comment (AleX83Xpert) [inline apps/condo/domains/resident/schema/FindOrganizationsByAddressService.js:17]: Please use case-sensitive mode for search-n-replace :)
```suggestion
} = require('@condo/domains/resident/utils/serverSchema/findOrganizationsByAddress')
```

File: apps/condo/domains/resident/utils/serverSchema/index.js [hunk 2]
```diff
@@ -114,13 +114,13 @@ async function registerResidentInvoice (context, data) {
     })
 }
 
-async function findOrganizationsForAddress (context, data) {
+async function FindOrganizationsByAddress (context, data) {
```
Comment (AleX83Xpert) [inline apps/condo/domains/resident/utils/serverSchema/index.js:117]:
```suggestion
async function findOrganizationsByAddress (context, data) {
```

File: apps/condo/domains/resident/utils/testSchema/index.js [hunk 3]
```diff
@@ -195,9 +195,9 @@ async function getResidentExistenceByPhoneAndAddressByTestClient (client, extraA
     return [data.result, attrs]
 }
 
-async function findOrganizationsForAddressByTestClient (client, attrs = {}) {
+async function FindOrganizationsByAddressByTestClient (client, attrs = {}) {
```
Comment (AleX83Xpert) [inline apps/condo/domains/resident/utils/testSchema/index.js:198]:
```suggestion
async function findOrganizationsByAddressByTestClient (client, attrs = {}) {
```

File: apps/condo/domains/resident/schema/FindOrganizationsByAddressService.js (hunk 1 above)
Comment (vovaaxeapolla) [reply apps/condo/domains/resident/schema/FindOrganizationsByAddressService.js]: Fixed it ty

**Review by AleX83Xpert — APPROVED** [review]

**Review by abshnko — APPROVED** [review]

**Pull request merged**